component/src/main/resources/META-INF/services/io.grpc.LoadBalancerProvider

```diff
@@ -0,0 +1,2 @@
+io.grpc.internal.PickFirstLoadBalancerProvider
```

Comment:
remove above binary files

Reply:
When I remove those binary files, I get an issue saying no functional service provider and try adding a dependency okhttp or netty